python/ray/serve/_private/api.py

```diff
@@ -175,7 +151,6 @@ def serve_start(
             " New http options will not be applied.",
         )
 
-        _check_http_options(client, http_options)
```

Comment:
Why is it okay to delete this? What are we replacing it with?

Reply:
Fair enough, but it seems like a useful warning, otherwise we'd be silently ignoring parameters and users might be confused.  Or is it no longer relevant because users don't call `serve.start()` anymore because it's deprecated?